Code/src/main/java/nl/utwente/group10/ui/exceptions/InvalidAnchorException.java

```diff
@@ -0,0 +1,5 @@
+package nl.utwente.group10.ui.exceptions;
+
+public class InvalidAnchorException extends RuntimeException {
```

Comment:
Nvm, As soon as I removed it I found where it was used (eclipse didnt find anything?)

It gets thrown in a connection if you feed a connectionAnchor that is neither an input nor an output anchor